modules/rostests/apitests/netapi32/DsRoleGetPrimaryDomainInformation.c

```diff
@@ -19,9 +20,19 @@ START_TEST(DsRoleGetPrimaryDomainInformation)
 
     // Get information about the domain membership of this computer.
     dwErrorCode = DsRoleGetPrimaryDomainInformation(NULL, DsRolePrimaryDomainInfoBasic, (PBYTE*)&pInfo);
-    ok(dwErrorCode == ERROR_SUCCESS, "DsRoleGetPrimaryDomainInformation returns %lu!\n", dwErrorCode);
-    ok(pInfo->MachineRole >= DsRole_RoleStandaloneWorkstation && pInfo->MachineRole <= DsRole_RolePrimaryDomainController, "pInfo->MachineRole is %u!\n", pInfo->MachineRole);
+    ok(dwErrorCode == ERROR_SUCCESS, "DsRoleGetPrimaryDomainInformation returned 0x%lx\n", dwErrorCode);
```

Comment:
Win32 errors are usually displayed in decimal. Revert please.

Reply:
Reverted to `%lu`.